packages/contracts/contracts/alice/bonds/DemoInvestmentWallet.sol

```diff
@@ -10,9 +10,16 @@ contract DemoInvestmentWallet is InvestmentWallet {
         InvestmentWallet(_projectCatalog) {
     }
 
-
     function requestTokens(DemoToken _token, uint _amount) public onlyOwner {
         _token.mint(_amount);
     }
 
+    function investAndRedeem(uint _amount, string memory _projectName) public onlyOwner {
```

Comment:
Hmm, why is this change in a dashboard PR?

Reply:
I think it was included from a previous commit by accident when switching branches
I'll remove this from the commit